Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/EDA_grades.ipynb

```diff
@@ -0,0 +1,2485 @@
+{
```

Comment:
<p>"Wyraźną zależność" to niekoniecznie... </p><p> </p><p>Wnioski są ciekawe, ale wykresy nie bardzo. Te kropki mało mówią o gęstości </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>